ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -389,30 +404,36 @@ extension ClientQueryViewController : OCQueryDelegate {
 		query.requestChangeSet(withFlags: OCQueryChangeSetRequestFlag(rawValue: 0)) { (_, changeSet) in
 			DispatchQueue.main.async {
 				self.items = changeSet?.queryResult
-				self.tableView.reloadData()
 
 				switch query.state {
-					case .contentsFromCache, .idle:
-						if self.items?.count == 0 {
-							self.message(show: true, imageName: "folder", title: "Empty folder".localized, message: "This folder contains no files or folders.".localized)
+				case .contentsFromCache, .idle:
+					if self.items?.count == 0 {
+						if self.searchController.searchBar.text != "" {
+							self.message(show: true, imageName: "folder", title: "No matches".localized, message: "There is no results for this search".localized)
```

Comment:
We should maybe use a different image for search. Do we have a SVG of a search icon somewhere else already @michaelstingl? One that we could reuse here to show when search didn't bring up any results?

Reply:
If you guys have the SVG file send it to me and I'll change it. Another option is to search for some SVG with public domain license.

Reply:
Core uses: 

![screenshot 2018-06-12 18 04 00](https://user-images.githubusercontent.com/2644445/41302638-4ac3707c-6e6b-11e8-8c88-16230f89ba56.png)

https://github.com/owncloud/core/blob/master/core/img/actions/search.svg

Reply:
So if everyone is fine with it, I'll use it then.

Reply:
👍

Reply:
Here it's the updated screen:
![image-1](https://user-images.githubusercontent.com/8929738/41338426-73e5cc34-6ef2-11e8-8d13-e71e421c4b9d.jpg)

Reply:
This is great! :)